src/patternfly/components/_all.scss

```diff
@@ -28,6 +28,8 @@
 @import "./Expandable/expandable.scss";
 @import "./FormControl/form-control.scss";
 @import "./Form/form.scss";
+
+// @import "./InlineEdit/inline-edit.scss";
```

Comment:
I think this needs to be uncommented?

Reply:
When the component is promoted from experimental, we add/uncomment the import. While still experimental, the compiled `.css` is called here https://github.com/patternfly/patternfly-next/blob/master/gatsby-browser.js

Reply:
I see, I guess I was confused because it's not listed in that "// Experimental components" section like the others. For some reason, gatsby was giving me a hard time linking the projects locally to test this out without uncommenting that line, so I'll just do this locally while we work through the React implementation.